src/plugins/intel_gpu/src/kernel_selector/cl_kernels/softmax_gpu_bf.cl

```diff
@@ -45,6 +45,16 @@ KERNEL (softmax_gpu_continuous_bfyx)(
     const uint items_num = data_set_size>>power;
     const uint leftovers = data_set_size-(items_num<<power);
 #endif
+    const uint data_set_idx_is_odd = data_set_idx - ((data_set_idx>>1)<<1) ;
```

Comment:
If the type is float, it will be aligned to 4 bytes every time, even though it is not odd. How about handle both cases?
Also I'd like to suggest to make a variable name more about the purpose.

Reply:
Okay, I see. I will update it. thanks.

Reply:
Fixed it. thanks.